orttraining/orttraining/core/graph/optimizer_builder.h

```diff
@@ -18,36 +18,73 @@ const std::vector<std::string> MOMENTS_PREFIXES({"Moment_1", "Moment_2"});
 const std::string LAMB_STEP_TENSOR_NAME = "Step";
 const std::string ADAM_UC_PREFIX = "Update_Count";
 
+namespace training_internal {
+extern const int64_t single_span_element;
```

Comment:
Why not directly define it as constexpr at here? Why do you split the definition to a separated cpp file?

Reply:
I wanted to have a guaranteed single instance of this constant.